src/getHumanAge.test.js

```diff
@@ -7,4 +7,21 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [1, 1] if animalAge < first + second', () => {
+    expect(getHumanAge(20, 20)).toEqual([1, 1]);
+  });
+
+  test('should return [0, 0] if animalAge === 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [2, 2] if animalAge > first + second', () => {
+    expect(getHumanAge(26, 26)).toEqual([2, 2]);
```

Comment:
The test description mentions 'animalAge > first + second', but the test case uses 'getHumanAge(26, 26)'. Ensure that the function logic and test case align with the intended condition.